mars/document_definition_scoring.py

```diff
@@ -21,7 +21,9 @@ def document_definition_scoring(key_min: int, key_max: int,
     @param key_max: highest document key
     @param path_to_model: path to folder with definition model
     """
+    path_to_tokenizer = config.models_dir + '/tokenizers/' + path_to_model
```

Comment:
Czemu nie os.path.join ?

Reply:
To chyba w większości kwestia przyzwyczajenia. Ja wolę dodawanie, bo z mojej perspektywy jest bezpieczniejsze. Dla  przykładu to nie jest takie oczywste, gdy się nie przeczyta dokumentacji, że `os.path.join('/foo/', '/bar') = '/bar'`. Gdybyśmy tu podawali jakieś specjalne obiekty, które nadpisują metodę `__fspath__` to oczywiście os.path.join wygrywa. Natomiast w tym wypadku dodawanie wystatczy .